src/lib/components/01-elements/link/link.tsx

```diff
@@ -21,7 +21,7 @@ export type LinkProps = AnchorHTMLAttributes<HTMLAnchorElement> & {
 };
 
 const colorLinkSchema = {
-  black: 'text-black hover:text-teal',
+  black: 'text-black hover:text-blue',
```

Comment:
@gstvdesign just to double check if this change is correct?

Reply:
Confirmed with Victor. Black links should always be blue when hovered.